container_images/autoversioner/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
+
+WORKDIR /workspace
+ADD gopath/bin /
```

Comment:
can we copy individually, instead, to be more explicit? I also like to copy the dockerfile into the images we produce so you can see how it was produced

Reply:
done.